nilearn/plotting/js_plotting_utils.py

```diff
@@ -189,23 +189,30 @@ def remove_temp_file(self):
         self._temp_file = None
 
 
-def colorscale(cmap, values, threshold=None, symmetric_cmap=True, vmax=None):
+def colorscale(cmap, values, threshold=None, symmetric_cmap=True,
+               vmax=None, vmin=None):
```

Comment:
We should complete the docstring here with parameters and return data.
Also gives the new push we need for triggering Appveyor.